spec/javascript/components/organisms/__snapshots__/SimpleModal.test.js.snap

```diff
@@ -24,7 +24,7 @@ exports[`SimpleModal it should render status type successfully 1`] = `
     isOpen={true}
   >
     <Modal
-      ariaHideApp={true}
+      ariaHideApp={false}
```

Comment:
what does this feature do, and what's the warning that's coming out of it? Our accessibility isn't great on the app, but I don't want to decrease accessibility if that's what this change does.

Reply:
This is in place only for the test environment because the `react-modal` package depends on the `setAppElement` method for a root or it will throw an error, so in the custom modal component that field is set to false only for the test environment to get around the fact that we are mounting the components directly, it should not impact the accessibility of the modal. It is the recommended work around for testing components with `react-modal`